tools/explorer/mw/action.go

```diff
@@ -28,6 +28,7 @@ func AsHandlerFunc(a Action) http.HandlerFunc {
 		object, result := a(r)
 
 		w.Header().Set("Content-Type", "application/json")
+		w.Header().Set("access-control-expose-headers", "pages")
```

Comment:
What is this for ?

Reply:
This enables axios to pick up the pages field in the response headers of the api calls.